cli/src/semgrep/rule_match.py

```diff
@@ -118,6 +121,13 @@ def start(self) -> core.Position:
     def end(self) -> core.Position:
         return self.match.location.end
 
+    def get_individual_line(self, line_number: int) -> str:
```

Comment:
Since this doesn't do any bounds checking does it need different type signature? 
```suggestion
    def get_individual_line(self, line_number: int) -> str | None:
```

Reply:
I don't think so? The old behavior, which I've preserved, just returns an empty string if the line was out of range 🤷 Or are you suggesting a more comprehensive change to that behavior?

Reply:
I think I'm still in ruby land too much of the time, looks like what would return null there raises an error here so this is fine.

```
>>> lines = []
>>> lines[0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```